### PR TITLE
Frontend authorization functional

### PR DESCRIPTION
**Pull request by Dkellam9**
fixes for devise register, login, updated NavBar to make sense.

### CODE REVIEW

**Comment by Dkellam9** [comment]
OwO

File: src/sharedComponents/Header.js [hunk 1]
```diff
@@ -18,16 +24,34 @@ class Header extends Component {
...
         </Nav>
-        <Nav className="pull-right">
+
+        {this.determineButton()}
+
+      </Navbar>
+    );
+  }
+
+  determineButton = () => {
```
Comment (winescout) [inline src/sharedComponents/Header.js]: determineButton could be named more informatively.  "loginLogoutButton", "userAuthButton" maybe?

File: src/services/AuthService.js [hunk 2]
```diff
@@ -2,31 +2,34 @@ import decode from 'jwt-decode'
...
 			console.log(token);
 			this.setToken(token)
 			//return json from response
+			console.log(statusResponse);
 			return statusResponse.json()
 		})
 	}
 
 	register = (user) => {
+		console.log('HEREEEEEEEEEE - AuthService user', user)
```
Comment (bwanicur) [inline src/services/AuthService.js]: For WIP console.log is ok, but I'd be really careful.  Very easy to forget about these.

File: src/pages/Users/Login.js [hunk 3]
```diff
@@ -17,23 +17,19 @@ class Login extends Component {
...
   render() {
-    let { email, password } = this.state.form.user
       return (
           <div>
             <div className="loginMain" >
               <form onSubmit={this.onSubmit}>
               <input
                 type="email"
                 name="email"
-                value={email}
```
Comment (winescout) [inline src/pages/Users/Login.js]: Why this change?  Its better to have your inputs be controlled by React, meaning that you set the value of the input explicitly as this code was doing.

File: src/pages/Users/Login.js [hunk 4]
```diff
@@ -17,23 +17,19 @@ class Login extends Component {
...
                 placeholder="Enter Email Here"
                 onChange={this.onChange}
               />
-              <nbsp />
               <input
                 type="password"
                 name="password"
-                value={password}
```
Comment (winescout) [inline src/pages/Users/Login.js]: same comment as above.  I'd rather see this line left in.

File: src/pages/Users/Login.js (hunk 3 above)
Comment (Dkellam9) [reply src/pages/Users/Login.js]: Something changed while hacking through a functioning login that I believed to be broken. It works, but it may be bad convention. Should I change?

File: src/pages/Users/Login.js (hunk 4 above)
Comment (Dkellam9) [reply src/pages/Users/Login.js]: "
